src/doc/data_into_visit/SiloFormat.rst

```diff
@@ -1,6 +1,1621 @@
 .. _Data_Into_VisIt_Silo:
 
-The Silo_ file format
-=====================
+The Silo file format
+====================
+
+If you are writing a conversion utility or if you have a simulation code written in C, C++, or Fortran then writing out Silo_ files is a good choice for getting your data into VisIt_.
+This section will illustrate how to use the Silo_ library to write out various types of scientific data.
+Since the Silo_ library provides bindings for multiple languages, including C, Fortran, and Python, the source code examples that demonstrate a particular topic will be given in more than 1 programming language, when appropriate.
+One goal of this section is to provide examples that are complete enough so that they can be readily adapted into working source code.
+This section will not necessarily explain all of the various arguments to function calls in the Silo_ library.
+You can refer to the `Silo Manual <https://software.llnl.gov/Silo/manual.html>`_ for more information.
+
+Using the Silo library
+~~~~~~~~~~~~~~~~~~~~~~
+
+VisIt_ is always built with support for reading Silo_ databases so Silo_ can be a good file format in which to store your data.
+This subsection includes information about using Silo_ such as including the appropriate header files and linking with the Silo_ library.
+
+Including Silo
+""""""""""""""
+
+When using any library in a program, you must tell the compiler about the symbols provided by the library.
+Here is what you need to include in your source code in order to use Silo:
+
+.. tabs::
+
+  .. code-tab:: c
+
+    #include <silo.h>
+
+  .. code-tab:: fortranfixed
+  
+    include "silo.inc"
+
+
+Linking with Silo
+"""""""""""""""""
+
+Before you can build a program that uses Silo_, you must locate the Silo_ include files and the Silo_ library.
+Silo_ is not distributed as part of the VisIt_ source code or binary installations so you must obtain it separately unless you are developing on the Windows platform.
+A link to the most up-to-date version of the Silo_ library's source code can be found on `Silo's Github site <https://github.com/LLNL/Silo/releases>`_.
+
+Once you download the Silo_ source code, building and installing it is usually only a matter of running its *configure* script and running *make*.
+You can even use the build_visit script from the `VisIt Web site <https://github.com/visit-dav/visit/releases>`_ to build Silo_ with support for HDF5.
+An example command line to build Silo_ with support for HDF5 is:
+
+.. code-block:: bash
+
+  ./build_visit --console --no-visit --no-thirdparty \
+  --thirdparty-path /usr/local \
+  --silo --hdf5 --szip
+
```

Comment:
```suggestion

Although Silo_ does not *require* HDF5, it is best to build Silo_ with it because the HDF5 driver supports more features than Silo_'s default PDB driver.
```